modules/custom/az_multilingual/az_multilingual.module

```diff
@@ -0,0 +1,28 @@
+<?php
+
+/**
+ * @file
+ * Contains az_multilingual.module.
+ */
+
+/**
+ * Implements hook_preprocess_HOOK() for links--language-block templates.
+ *
+ * Override template to include Bootstrap .btn classes on rendered links.
+ */
+function az_multilingual_preprocess_links__language_block(&$variables) {
+
+  $links = $variables['links'];
+  $lang = \Drupal::languageManager()->getCurrentLanguage()->getId();
+
+  if (!empty($links)) {
+    foreach ($links as $key => $link) {
+
+      // Include .active class for Bootstrap styling (Drupal uses .is-active)
+      if (!empty($lang) && !empty($links[$lang])) {
+        $variables['links'][$lang]['link']['#options']['attributes']['class'][] = 'active';
+      }
```

Comment:
This section can be moved outside the `foreach` loop I think as it only depends on `$lang`